data/mods/ssb/conditions.ts

```diff
@@ -72,6 +72,20 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			}
 		},
 	},
+	blitz: {
```

Comment:
```suggestion
	blitzuser: {
```
the blitz rule got ya there